src/filtering/convert.rs

```diff
@@ -423,6 +431,22 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_filetype_with_comments() -> Result<()> {
```

Comment:
also, it makes sense to write a test to check the file without comments too

Reply:
It already exists above